src/Avalonia.Visuals/Media/DrawingContext.cs

```diff
@@ -154,20 +178,6 @@ public void DrawText(IBrush foreground, Point origin, FormattedText text)
             }
         }
 
-        /// <summary>
-        /// Draws a filled rectangle.
-        /// </summary>
-        /// <param name="brush">The brush.</param>
-        /// <param name="rect">The rectangle bounds.</param>
-        /// <param name="cornerRadius">The corner radius.</param>
-        public void FillRectangle(IBrush brush, Rect rect, float cornerRadius = 0.0f)
```

Comment:
We shouldn't break backwards compatibility in our user-facing DrawingContext. Consider keeping previous versions.